package.json

```diff
@@ -108,6 +107,8 @@
 		"npm": "6.11.3"
 	},
 	"dependencies": {
+		"@wordpress/components": "8.5.0",
```

Comment:
I think we have several dependencies listed in `devDependencies` that actually belong to `dependencies`. I moved `@wordpress/components` in this PR, but if it looks good I think we could move several more.

Reply:
Yup agreed, anything that is used in production builds _directly_ (as opposed to just being included for tests) should be moved into `dependencies`.

Reply:
Created an issue to handle that: #1424.